classes/class-blog-handler.php

```diff
@@ -280,6 +281,9 @@ public function get_friendly_path( $path ) {
 			$path = $this->append_subdirectory_install_base_path( $path );
 		}
 
+		// Make sure path isn't too big for DB Column.
```

Comment:
I am wondering whether truncating the name at this point would cause any duplicate issues?

Reply:
It does have the potential, but I think it is highly unlikely. It would have be nice just to updated the size of the path column in wordpress, but since path is stored in a core table this is not possible. This is the best solution for the time being.

Reply:
Okay thanks